playhand.js

```diff
@@ -306,21 +306,48 @@ function postflop(cardsString, boardCardsString, playersInHand, potSizeAtStartOf
                 raiseTo: 7
             })
         } else if (usedHoleCards.length === 1) {
-            // TODO: it matters what card we have
+            if (["A", "K"].includes(usedHoleCards[0].rank)) {
+                betOptions.push({
+                    message: "Flush using 1 hole card (A or K)",
+                    callTo: mb.ALL,
+                    raiseTo: 4
+                })
+            }
+
             betOptions.push({
-                message: "Flush using 1 hole card",
-                callTo: 10,
+                message: "Flush using 1 hole card (not A or K)",
+                callTo: 5,
                 raiseTo: 0
             })
         }
+
         // Flush on the board
-        // TODO: We might beat the board
-        // Right now this is just for logging
-        betOptions.push({
-            message: "Flush on the board",
-            callTo: 0,
-            raiseTo: 0
+
+        // Check if we beat the board and have A or K
+        const ranksInSuit = handCards.map((card) => {
+
+            // Can look at any board card because we know all 5 are the same suit
+            if (card.suit === boardCards[0].suit) {
+                return card.rank
+            }
+
+            return ""
         })
+
```

Comment:
Alternate proposal:

```js
const hasHighCardInSuit = handCards.some(card => ["A", "K"].includes(card.rank) && card.suit === boardCards[0].suit)
```
and then use that boolean to decide if you should bet or call/check